CMakeLists.txt

```diff
@@ -1,73 +1,72 @@
-cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.16 FATAL_ERROR) # Policies <= CMP0097 default to NEW
 
-message(AUTHOR_WARNING "If the build fails, please try the autotools/qmake method.")
-
-if (POLICY CMP0074)
-    cmake_policy(SET CMP0074 NEW)
-endif()
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules)
```

Comment:
This is unsafe, `list(APPEND` is the right choice. 

It is unsafe because you need to prefer system-supplied modules, because distributions can patch them to account for distribution- or even site-specific peculiarities.

Reply:
@zeule 
Ah, I see.

Reply:
Fixed.

Reply:
>It is unsafe because you need to prefer system-supplied modules

Docs say that CMAKE_MODULE_PATH is `a search path for CMake modules to be loaded by the include() or find_package() commands before checking the default modules that come with CMake`. So the modules that come from there are always preferred against the ones that come with CMake.
Also CMake docs say `CMAKE_MODULE_PATH is empty by default and it  is intended to be set by project` so it is intended to set this list instead of append to it.
Or CMAKE_MODULE_PATH  is usually used in different way?